spec/requests/solidus_stripe/payment_intents_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'solidus_stripe_spec_helper'
+
+RSpec.describe SolidusStripe::PaymentIntentsController, type: :request do
```

Comment:
If I'm not mistaken here, the best practice is to avoid adding the `_controller` part to the filename and to the main describe name. In general, the file seems too tied to the controller, while requests specs should be more of a full request-cycle spec.

Reply:
I just checked the RSpec generator as I wasn't sure about what changes I added, this is how it looks like:

```rb
require 'rails_helper'

RSpec.describe "<%= class_name.pluralize %>", <%= type_metatag(:request) %> do
  describe "GET /<%= name.underscore.pluralize %>" do
    it "works! (now write some real specs)" do
      get <%= index_helper %>_path
      expect(response).to have_http_status(200)
    end
  end
end
```

So the only thing I changed was removing the quotes from around the class name so I could use `described_class`, happy to change it if there's a different/better way.

Reply:
Yes, but I think that the class should not be a controller in the request specs. Looking around for examples it seems like it's a more high level concept that a specific class.